src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbFilter/breadcrumbFilter.tsx

```diff
@@ -114,7 +123,7 @@ class BreadcrumbFilter extends React.Component<Props, State> {
 
     return (
       <Wrapper>
-        <DropdownControl menuWidth="20vh" blendWithActor button={this.getDropDownButton}>
+        <DropdownControl menuWidth="25vh" blendWithActor button={this.getDropDownButton}>
```

Comment:
It seems odd to me that width is defined using `vh`, which is a unit for height.

https://developer.mozilla.org/en-US/docs/Web/CSS/length

Reply:
oh I don't like it too... I believe I've copied it from the component savedSearchSelector
![image](https://user-images.githubusercontent.com/29228205/81172389-6c221880-8f9e-11ea-87ed-350226fcecb7.png) but I was planning to update it too..